tests/unit/test_charm.py

```diff
@@ -119,8 +104,6 @@ def test_remote_write_configuration(self, mock_push: MagicMock, mock_restart: Ma
             {"remote_write": json.dumps({"url": "http://1.1.1.2:9090/api/v1/write"})},
         )
 
-        path, content = mock_push.call_args[0]
-        content = yaml.safe_load(content)
         expected_config: Dict[str, Any] = {
             "integrations": {
```

Comment:
I think it might make sense to use a rewrite_rule to change the `job_name` label, which currently has the `integrations/agent` value, to be in line with those we use elsewhere based on topology:
![Screenshot from 2022-04-28 18-14-16](https://user-images.githubusercontent.com/14020598/165798029-4ec7bd28-e344-4a11-8677-3d2c49f08d39.png)

Reply:
Addressed in 073a3afdc44640a180fca43dde95178a44041dcd